pkg/controller/provider/ova-setup.go

```diff
@@ -193,6 +193,8 @@ func (r *Reconciler) makeOvaProviderPodSpec(pvcName string, providerName string)
 
 	nfsVolumeName := fmt.Sprintf("%s-%s", nfsVolumeNamePrefix, providerName)
 	ovaContainerName := fmt.Sprintf("%s-pod-%s", ovaServer, providerName)
+	allowPrivilegeEscalation := false
+	runAsNonRoot := true
```

Comment:
how about aligning this with other places by renaming this variable to nonRoot?
```suggestion
	nonRoot := true
```